plasmapy/physics/distribution.py

```diff
@@ -70,25 +77,286 @@ def Maxwellian_1D(v: units.m/units.s,
+                           vz_drift=0*units.m/units.s):
+    r"""Return the probability of finding a particle with velocity components
+    `v_x`, `v_y`, and `v_z`in m/s in an equilibrium plasma of temperature 
+    `T` which follows the 3D Maxwellian distribution function. This 
+    function assumes Cartesian coordinates.
+
+    Parameters
+    ----------
+    vx: Quantity
+        The velocity in x-direction units convertible to m/s
+        
+    vy: Quantity
+        The velocity in y-direction units convertible to m/s
+        
+    vz: Quantity
+        The velocity in z-direction units convertible to m/s
+
+    T: Quantity
+        The temperature, preferably in Kelvin
+
+    particle: string, optional
+        Representation of the particle species(e.g., 'p' for protons, 'D+'
+        for deuterium, or 'He-4 +1' for $He_4^{+1}$ : singly ionized helium-4),
+        which defaults to electrons.
+        
+    vx_drift: Quantity
+        The drift velocity in x-direction units convertible to m/s
+        
+    vy_drift: Quantity
+        The drift velocity in y-direction units convertible to m/s
+        
+    vz_drift: Quantity
+        The drift velocity in z-direction units convertible to m/s
+
+    Returns
+    -------
+    f : Quantity
+        probability in Velocity^-1, normalized so that:
+        $\iiint_{0}^{\infty} f(\vec{v}) d\vec{v} = 1}
 
-    if T.unit.is_equivalent(units.K):
-        T = T.to(units.K, equivalencies=units.temperature_energy())
+    Raises
+    ------
+    TypeError
+        The parameter arguments are not Quantities and
+        cannot be converted into Quantities.
 
-    # Get mass
+    UnitConversionError
+        If the parameters is not in appropriate units.
 
-    if is_electron(particle):
-        m_s = m_e
+    ValueError
+        If the temperature is negative, or the particle mass or charge state
+        cannot be found.
 
-    else:
-        try:
-            m_s = ion_mass(particle)
-        except Exception:
-            raise ValueError("Unable to find ion mass in Maxwellian_1D")
+    Notes
+    -----
+    In one dimension, the Maxwellian speed distribution function describing
+    the distribution of particles with speed v in a plasma with temperature T
+    is given by:
 
-    T = k_B*T
+    .. math::
+    f = (\pi * v_Th^2)^{-3/2} \exp(-(\vec{v} - \vec{v_{drift}})^2 / v_Th^2)
+    where v_Th = \sqrt(2 k_B T / m) is the thermal speed
 
-    f = np.sqrt((m_s/(2*pi*T)))*np.exp(-m_s/(2*T)*(v-V_drift)**2)
+    See also
+    --------
+    Maxwellian_1D
+
+    Example
+    -------
+    
+    
+    """
+    # convert temperature to Kelvins
+    T = T.to(units.K, equivalencies=units.temperature_energy())
+    # get thermal velocity and thermal velocity squared
+    vTh = thermal_speed(T, particle=particle, method="most_probable")
+    # accounting for thermal velocity in 3D
+    vTh3D = np.sqrt(3) * vTh
+    vThSq = vTh3D ** 2
+    # Get particle velocity squared
```

Comment:
This isn't a big deal, but, "relative" particle velocity, maybe, given it's shifted by the drift one?